test/infra/README.md

```diff
@@ -0,0 +1,74 @@
+## TODO: Add architecture diagram
+
+# Set up Config Connector via Anthos Config Management
```

Comment:
This is instructions for 1 time setup of the cluster that we submit integration tests to, and the integration tests will themselves _create_ separate k8s clusters right?

A little "overview" explaining that would be helpful (or correcting me).

Reply:
Done